### PR TITLE
Adds a foam armblade

### PR DESCRIPTION
**Pull request by kol1th**
As the title implies, adds (ports) a foam armblade (from tg). The in-hand sprite is the same as the regular armblade, and you can get it as an arcade machine prize. 
image

### CODE REVIEW

**Comment by Konater** [comment]
What's an armblade?

**Comment by ComicIronic** [comment]
About 180 pounds.

File: html/changelogs/kolith.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: kolith
+delete-after: True
+changes: 
+- rscadd: Adds a foam armblade toy. It can be aquired as a prize from an arcade machine.
```
Comment (JustSumBody) [inline html/changelogs/kolith.yml]: Should be acquired

**Comment by kol1th** [comment]
Wew. I'll fix it when I get home tomorrow.

**Comment by kol1th** [comment]
Shit. edit: let's pretend that didn't happen

**Pull request merged**